dev/build_release.py

```diff
@@ -222,6 +222,9 @@ def __debian_build(cls, name, options, build_number, gradle_root):
       '-PbintrayUser="{user}"'.format(user=bintray_user)
     ]
 
+    if os.path.exists('maven-override.gradle'):
```

Comment:
I think this should be controlled by a flag. Otherwise the override is sticky (earlier runs with the override will inadvertantly affect the later runs). The flag can be the name of the file. Setting it enables the override.

It's not clear to me how this is used and whether this is a standard name or not.